minject/registry.py

```diff
@@ -86,9 +102,35 @@ def config(self) -> RegistryConfigWrapper:
     def resolve(self, key: "RegistryKey[T]") -> T:
         return self[key]
 
+    async def _aresolve(self, key: "RegistryKey[T]") -> T:
```

Comment:
I named this `_aresolve` rather than `aresolve` as I didn't think `aresolve` should be included in public API (only use `aget`). This unfortunately has the down side of the sync versions of this and a related method and the async versions having different names.

Let me know if we thing `_resolve` should be renamed to `_resolve_resolvable` to match `_aresolve_resolvable`.

Reply:
I think `_resolve` should go away entirely — it's unnecessary indirection. 😅 

(See https://github.com/duolingo/minject/pull/52.)